doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -208,7 +208,7 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.ntw.11` | Network | The networking solution **should** be able to be centrally administrated and configured. |
 | `req.inf.ntw.14` | Network | The Architecture **must** support capabilities for integrating SDN controllers. |
 | `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
-| `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Container Network Interface (CNI). |
+| `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple network interfaces. |
```

Comment:
Agree with the sentiment however suggested wording isn't clear if we're talking about for Pods or Nodes. 

Suggest changing "network interfaces" to "connection points" in line with [chapter 4 from the Reference Model](../../blob/master/doc/ref_model/chapters/chapter04.md) and specifying that it's for Pods.

Reply:
Also, fix small typo.
`co-exisence` -> `coexistence`

Reply:
I've updated both. Can you please both review the changes?

Reply:
Thanks. Good from my side.

Reply:
Yes me too, thanks